core/templates/dev/head/pages/topic-viewer-page/topic-viewer-page.directive.html

```diff
@@ -13,6 +13,11 @@
         <a class="oppia-topic-viewer-tabs-text" ng-click="$ctrl.setActiveTab('practice')" translate="I18N_TOPIC_VIEWER_PRACTICE">
         </a>
       </span>
+      <span ng-class="{'oppia-topic-viewer-tabs-active': $ctrl.activeTab === 'skill'}">
+        <a class="oppia-topic-viewer-tabs-text" ng-click="$ctrl.setActiveTab('skill')">
+          Skills
```

Comment:
This should be added as a constant to the translation files (en.json and qqq.json).

Reply:
This part is removed.